src/ts/scene/ShopScene.ts

```diff
@@ -36,25 +54,32 @@ export default class ShopScene extends Scene
                 this.scale.startFullscreen();
             }
         })
+
+        this.load.image(ShopScene.KEY.IMAGE.ShopOwner,"assets/images/shopScene/ShopOwner.png");
+        this.load.image(ShopScene.KEY.IMAGE.ShopPaper,"assets/images/shopScene/paper-plain.png");
+        this.load.image(ShopScene.KEY.IMAGE.ShopMyDec,"assets/images/shopScene/MYDEC.png");
+        this.load.image(ShopScene.KEY.IMAGE.ShopExit,"assets/images/shopScene/EXIT.png");
+
+        /** 더미 카드 이미지 */
+        this.load.image(ShopScene.KEY.IMAGE.ShopPotion,"assets/iamges/shopScene/potion.png");
     }
 
     create(): void
     {
```

Comment:
셔플 메소드를 지역변수로 선언하여 캡슐화하고, 타입을 명확히 지정해준다면 컴파일 단계에서 오류를 검출할 수 있게 됩니다.
```suggestion
    {
        const Shuffle: (array: string[]) => string[] = Phaser.Utils.Array.Shuffle;
```